lib/xdr/accounts/thresholds.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Stellar.XDR.Thresholds do
+  @moduledoc """
+  Representation of an AccountEntry's Thresholds.
+  Thresholds stores unsigned bytes: [weight of master|low|med|high]
+  """
+  alias Stellar.XDR.Opaque4
+
+  @behaviour XDR.Declaration
+
+  @type thresholds :: [
+          master_weight: non_neg_integer(),
+          low: non_neg_integer(),
+          med: non_neg_integer(),
+          high: non_neg_integer()
+        ]
+
+  @type t :: %__MODULE__{thresholds: thresholds}
+
+  defstruct [:thresholds]
+
+  @spec new(master_weight: byte(), low: byte(), med: byte(), high: byte()) :: t()
+  def new([master_weight: _master_weight, low: _low, med: _med, high: _high] = thresholds),
```

Comment:
@JuanCamiloPerilla I suggest adding another function clause where an exception is raised if the tresholds list is not correct.